relay-server/src/services/buffer/envelope_repository/mod.rs

```diff
@@ -0,0 +1,104 @@
+use crate::services::buffer::common::{EnvelopeBufferError, ProjectKeyPair};
+use crate::services::buffer::envelope_repository::memory::MemoryEnvelopeRepository;
+use crate::services::buffer::envelope_repository::sqlite::SqliteEnvelopeRepository;
+use crate::{Envelope, MemoryChecker};
+use hashbrown::HashSet;
+use relay_config::Config;
+
+pub mod memory;
+pub mod sqlite;
```

Comment:
I believe these could be private.

Reply:
Only one, could, since we use the `SqliteEnvelopeRepository` in benchmarks.